www/pages/project-advanced.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+permalink: /project-advanced
+title: Transfer or delete a project
+---
+
+To access the advanced project settings:
+
+- Select **SETTINGS** from above the project files
+- At the bottom of the page, you have three options:
+  - Public/private setting: to make your project public or private
+  - **Transfer project**: this option transfers the ownership of your project to another Mergin user or organisation.
+  - **Delete project**: to delete the project and all the data
```

Comment:
Worth adding a warning this action is not undoable

Reply:
Technically it is!